README.md

```diff
@@ -21,3 +21,22 @@ You will get a dump of:
 
 In order to get data about other parts of the cluster (not specific to KubeVirt) you should
 run `oc adm must-gather` (without passing a custom image). Run `oc adm must-gather -h` to see more options.
+
+### Configuring image registries for must-gather
+
+The must-gather image created a node-gather deamon-set during execution.
+This daemon-set pulls the node-gather image.
+The registry for this image can either be defined during build time, or skipped.
+In the second case the image lookup relies on resolution by search registries.
+
+To build the image specifying a registry include a IMAGE_REGISTRY parameter when building
+must-gather:
+```ssh
+make build IMAGE_REGISTRY=<registry url>
+````
+
+To rely on search registries, update the "/etc/containers/registries.conf" file:
```

Comment:
shouldn't there be another steps inside the OCP cluster to support additional registries ? By looking at kubevirtci there is a different way to update it: 

https://github.com/kubevirt/kubevirtci/blob/963093ed0549bcff1b143e37b3a2799701789ac1/cluster-provision/okd/scripts/provision.sh#L158